provisioning/ci/README.md

```diff
@@ -0,0 +1,32 @@
+# CI provisioning
+
+### Prepare necessary AWS resources
+Run the `prepare-aws-resources.sh` script using `CI-Platform-Services-Team` account (use
+`Keboola-CI-Platform-Services-Team-AWSAdministratorAccess` profile). The script prepares resources needed for Terraform
+remote backend and user account to use in CI.
+
+### Prepare cloud resources
+Initialize & run Terraform to create resources for CI. Notice no `tfstate` file is created, it's actually automatically
+saved to the remote S3 bucket si it can be accessed by the CI pipeline later.
+
+
+```bash
+terraform init -var-file=../ci/terraform.tfvars -backend-config=../ci/s3.tfbackend 
+terraform apply -var-file=../ci/terraform.tfvars
+```
+
```

Comment:
Koukam, ze si tu vynechal sekci `Configure CI pipeline credentials`. Predpokladam, ze proto, ze tu neni doreseny pristup z Gihub Actions na Terraform state backend a infra, takze vystupy z TF si vzal a preklopil je do `.github/workflows/push.yml`? To by tu bylo IMHO dobry zminit.

Reply:
Jo, jak jsem psal martinovi - už jsem se nechtěl zavrtávat do další nory s OIDC, nicméně když jsem už upravoval ty resources, tak jsem to teda rovnou udělal v TF a správným accountu

Reply:
Tak pak dej vedet a a muzu pak pomoct odjebat ty klice a nastvit to at se pouzije oidc